backend/tests/reservation/getAll.js

```diff
@@ -0,0 +1,65 @@
+import chai, { assert } from 'chai';
```

Comment:
maybe there should be a short comment at top just saying which API in which file this tests, just to link tests to actual functionality?